step.yml

```diff
@@ -120,12 +120,14 @@ inputs:
       value_options:
         - "yes"
         - "no"
-  - wait_for_simulator_boot: "yes"
+  - headless_mode: "yes"
```

Comment:
please check if `headless_mode` works with older xcodes

Reply:
Checked  headless_mode with older xcodes:
The headless mode option for xcodebuild test is working only with >= Xcode 9.x.
http://shashikantjagtap.net/hands-xcuitest-features-xcode-9/

We stil have an older Xcode stack (Xcode 8.x)
But it's not a problem.
Because the Xcode 8 will start the simulator without any problem.
So the test will work perfectly but not in headless mode.